.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "YarnEditor"]
+	path = YarnEditor
+	url = https://github.com/TranquilMarmot/YarnEditor.git
```

Comment:
This will need to be updated once https://github.com/YarnSpinnerTool/YarnEditor/pull/186 is merged